pkg/api/controller.go

```diff
@@ -3006,6 +3011,66 @@ func (c *Controller) GetCurrentUser(w http.ResponseWriter, r *http.Request) {
 	writeResponse(w, http.StatusOK, response)
 }
 
+func (c *Controller) sendResetPasswordEmail(email []string, token string) error {
+	err := c.Emailer.SendEmail(email, token, token, []string{})
+	if err != nil {
+		c.Logger.
+			WithError(err).
+			WithFields(logging.Fields{
+				"email": email,
+			}).Debug("Error sending email")
+		return err
+	}
+	c.Logger.WithFields(logging.Fields{
+		"email": email,
+	}).Info("Succefully sent email")
```

Comment:
```suggestion
	}).Info("Successfully sent email")
```
Log messages are most useful if words in them are spelled consistently.  Perhaps the IDE check spelling on log messages?

Reply:
Fixed. Bad spelling on my side. (That's what happens when coding at 12 AM)